tripy/pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 requires-python = ">= 3.9"
 license = {text = "Apache 2.0"}
 dependencies = [
+  "tensorrt~=10.0",
```

Comment:
Need to decide if we want to pin to 10.1 or allow any 10.X.

Reply:
Resolving this - I think it's fine to keep it permissive for now. We can pin to a specific version later if we need to.